pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+You can attach existing clusters to Kommander whether or not those clusters have networking restrictions.
+
+For attaching existing clusters without networking restrictions, the requirements depend on the Kommander 1.X version that you plan to use. The supported Kubernetes versions are between version 1.16.x and 1.20.x with a maximum version of 1.20.5. To attach an existing cluster, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.
+
+For attaching clusters _with_ networking restrictions, the lowest Kubernetes version should be less than or equal to version 1.19.x. For these types of clusters, the Kommander cluster uses a tunnel to access the clusters you plan to attach. Any managed service you want to expose in the Kommander cluster requires the creation of a reverse proxy, or the use of HTTPS_PROXY environment variables that point to the Kommander tunnel proxy server.
```

Comment:
We need to use 'must' here, not 'should', when discussing requirements.  'Should' implies it might work, which is weasel wordy...  

Also, do we really mean 'less than 1.19.x'? Don't we mean 'greater than'?  This whole 'networking restrictions' feature is part of Kommander 1.4 which is released with Konvoy 1.8, which has a default Kubernetes version of 1.20.x, so it seems strange to me that we would say "less than 1.19.x" here.  Please confirm with engineering, but I think this should read:

```suggestion
For attaching clusters _with_ networking restrictions, the Kubernetes version of the cluster you wish to attach must be greater than or equal to version 1.19.x. For these types of clusters, the Kommander cluster uses a tunnel to access the clusters you plan to attach. Any managed service you want to expose in the Kommander cluster requires the creation of a reverse proxy, or the use of HTTPS_PROXY environment variables that point to the Kommander tunnel proxy server.
```

Reply:
Committing this - getting this in now.

Reply:
Resolved as noted.